extensions/amp-ad-network-doubleclick-impl/0.1/doubleclick-a4a-config.js

```diff
@@ -193,9 +196,23 @@ export class DoubleclickA4aEligibility {
       addExperimentIdToElement(experimentId, element);
       forceExperimentBranch(win, DOUBLECLICK_A4A_EXPERIMENT_NAME, experimentId);
     }
-    return ![DOUBLECLICK_EXPERIMENT_FEATURE.REMOTE_HTML_CONTROL,
-      DOUBLECLICK_EXPERIMENT_FEATURE.USDRUD_CONTROL].includes(
-        experimentId);
+    // If we are in the Remote.html or USDRUD controls, then we are supposed
+    // to use Delayed Fetch in accordance with the existing carveouts.
+    switch (experimentId) {
+      case DOUBLECLICK_EXPERIMENT_FEATURE.REMOTE_HTML_CONTROL:
```

Comment:
You don't actually care about the controls here so you can just make this:
switch (experimentId) {
    case DOUBLECLICK_EXPERIMENT_FEATURE.LAUNCH_EXPERIMENT:
       return true;
    case DOUBLECLICK_EXPERIMENT_FEATURE.USDRUD_EXPERIMENT:
       return !useRemoteHtml;
    case DOUBLECLICK_EXPERIMENT_FEATURE.REMOTE_HTML_EXPERIMENT:
       return !hasUSDRD;
}
return !hasUSDRD && !useRemoteHtml;

Reply:
done, actually had to toss in an RTC check at the end too